README.md

```diff
@@ -35,7 +35,9 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+let andy = Instructor.findOne({name: "Andy"})
+andy.wishlist_items.save({description: "Resin Laying Deer Figurine, Gold"})
```

Comment:
Close.

We don't get crud methods on our subdocument arrays like we did in rails. To save new `wishlist_items` on the `andy` object, we first push onto the `wishlist_items` array and then call save on `andy`:
```js
// Assuming we have model for wishlist_items, named WishItem
Instructor.findOne({name: "Andy"}, (andy)=>{
    andy.wishlist_items.push(new WishItem({description: "Resin Laying Deer Figurine, Gold"}))
    andy.save();
}
```